server/database/connection.js

```diff
@@ -0,0 +1,22 @@
+const { Pool } = require('pg');
+const url = require('url');
+require('dotenv').config();
+
+let dbUrl = process.env.DATABASE_URL;
+if (process.env.NODE_ENV === 'dbtest' || process.env.NODE_ENV === 'test') {
```

Comment:
dbtest is not a node environment variable you have set in the package.json file. Generally, we need only one environment for testing anyway, so we probably won't need to check it.
```suggestion
if (process.env.NODE_ENV === 'test') {
```